doc/source/interval_exchanges/index.rst

```diff
@@ -23,3 +23,4 @@ Interval exchange transformations
    iet
    integer_iet
    iet_family
+   flip_sequence
```

Comment:
This was not referenced anywhere. Should it be added here?

Reply:
Yes. Thanks.